main.go

```diff
@@ -357,7 +361,52 @@ func notifyWaiters(w http.ResponseWriter, r *http.Request) {
 }
 
 func main() {
+	log.SetFlags(log.Lshortfile | log.Ldate | log.Lmicroseconds)
```

Comment:
Why do we need this?

Reply:
not stricly necessary, but I prefer to see the microsecond in the logs, otherwise log.Printf only shows second resolution.
```
2024/05/05 22:59:39 Listening on :8080
```
vs
```
2024/05/05 23:00:14.173627 main.go:404: Listening on :8080
```
The log.Lshortfile could be removed, its useful for debugging occasionally, but in this case there is only one file anyway.

Reply:
Yeah I don't think we need so far. Let's not put SetFlags in this PR, and revisit this when necessary.

Reply:
removed